provider-utils/awscloudformation/default-values/livestream-defaults.json

```diff
@@ -4,24 +4,24 @@
     "gopSize": "1",
     "gopPerSegment": "2",
     "segsPerPlist": "3"
-    },
-    "mediaLive": {
-        "securityGroup": "0.0.0.0/0",
-        "ingestType": "RTMP_PUSH",
-        "encodingProfile": "FULL",
-        "autoStart": "YES"
-    },
-    "mediaPackage": {
-        "endpoints": "HLS,DASH",
-        "startOverWindow": "86400"
-    },
-    "mediaStorage": {
-        "storageType": "mStore"
-    },
-    "cloudFront": {
-        "enableDistrubtion": "YES",
-        "priceClass": "PriceClass_100",
-        "sBucketLogs": "",
-        "sLogPrefix": "cf_logs/"
-    }
+  },
+  "mediaLive": {
+    "securityGroup": "0.0.0.0/0",
+    "ingestType": "RTMP_PUSH",
+    "encodingProfile": "FULL",
+    "autoStart": "YES"
+  },
+  "mediaPackage": {
+    "endpoints": "HLS,DASH",
+    "startOverWindow": "86400"
+  },
+  "mediaStorage": {
```

Comment:
mediaStore would be a better name, consistent with service name (i know these edits came from linting, but  still trying to call out minor things)

Reply:
Probably best left for #1